src/main/java/com/softserveinc/ita/vsaroz/tests/RozetkaTest.java

```diff
@@ -0,0 +1,22 @@
+package com.softserveinc.ita.vsaroz.tests;
+
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+import com.softserveinc.ita.vsaroz.pageobjects.RozetkaPage;
+import com.softserveinc.ita.vsaroz.utils.runners.TestRunner;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.ITestResult;
+import org.testng.annotations.*;
+
+public class RozetkaTest extends TestRunner {
+
+    @Test
+    public void rozetkaTest() {
```

Comment:
what are you verifying here? change name of the test by the sense of verification

verify***Test <- naming of test methods

Reply:
renamed